nix/sources.json

```diff
@@ -1,148 +1,161 @@
-    },
-    "nodejs-headers": {
-        "builtin": false,
-        "rev": "v12.19.0",
-        "sha256": "1fb0dkzsprilybg30nhgdy0fcaqx9kc89iyrw6871frkaqmcdwwn",
-        "type": "file",
-        "url": "https://nodejs.org/download/release/v12.19.0/node-v12.19.0-headers.tar.gz",
-        "url_template": "https://nodejs.org/download/release/<rev>/node-<rev>-headers.tar.gz"
-    },
-    "pre-commit-hooks.nix": {
-        "branch": "master",
-        "builtin": false,
-        "description": "Seamless integration of https://pre-commit.com git hooks with Nix.",
-        "homepage": "",
-        "owner": "cachix",
-        "repo": "pre-commit-hooks.nix",
-        "rev": "431227b41d0aaa83fabdee6611e145b7d9bd150b",
-        "sha256": "0ff24n8i3mfw0jw5r5lsgvbahmh88zhi3qsk9yxrb8x82jprgzsy",
-        "type": "tarball",
-        "url": "https://github.com/cachix/pre-commit-hooks.nix/archive/431227b41d0aaa83fabdee6611e145b7d9bd150b.tar.gz",
-        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
-    },
-    "sphinxcontrib-haddock": {
-        "branch": "master",
-        "builtin": false,
-        "description": "Tools for using Haddock with Sphinx",
-        "homepage": null,
-        "owner": "michaelpj",
-        "repo": "sphinxcontrib-haddock",
-        "rev": "f3956b3256962b2d27d5a4e96edb7951acf5de34",
-        "sha256": "0mlxa5zzgdka0c6sj7pp4cfvc8rcrd8g2z2113vp614l43g17miv",
-        "type": "tarball",
-        "url": "https://github.com/michaelpj/sphinxcontrib-haddock/archive/f3956b3256962b2d27d5a4e96edb7951acf5de34.tar.gz",
-        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
-    },
-    "stackage.nix": {
-        "branch": "master",
-        "description": "Automatically generated Nix expressions of Stackage snapshots",
-        "homepage": "",
-        "owner": "input-output-hk",
-        "repo": "stackage.nix",
-        "rev": "e32c8b06d56954865725514ce0d98d5d1867e43a",
-        "sha256": "18dpgyk5hrc6w3f9nwsb22k5ar9q5c8x40mszxzqmrdhkrymrykk",
-        "type": "tarball",
-        "url": "https://github.com/input-output-hk/stackage.nix/archive/e32c8b06d56954865725514ce0d98d5d1867e43a.tar.gz",
-        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
-    },
-    "thorp": {
-        "branch": "master",
-        "description": "Synchronisation of files with the cloud using the hash of the file contents",
-        "homepage": "https://github.com/kemitix/thorp",
-        "owner": "kemitix",
-        "repo": "thorp",
-        "rev": "42634f9c09664ff9d5981fbd1212fe3b9b419afc",
-        "sha256": "0bxh4b3kkpm5ls3v3329gd3zljzfph5315z8plvi2ac8y2y10bs6",
-        "type": "tarball",
-        "url": "https://github.com/kemitix/thorp/archive/42634f9c09664ff9d5981fbd1212fe3b9b419afc.tar.gz",
-        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
-    }
+  "easy-purescript-nix": {
```

Comment:
Looks like your JSON auto-formatter is going strong :)

Reply:
noooooooo :sweat_smile: